dwl.c

```diff
@@ -1724,6 +1728,13 @@ xwaylandready(struct wl_listener *listener, void *data) {
 	netatom[NetWMWindowTypeUtility] = getatom(xc, "_NET_WM_WINDOW_TYPE_TOOLBAR");
 	netatom[NetWMWindowTypeToolbar] = getatom(xc, "_NET_WM_WINDOW_TYPE_UTILITY");
 
+	for (i = 0; i < NetLast; i++) {
+		if (!netatom[i]) {
+			fprintf(stderr, "getatom failed for netatom %ld", i);
+			return;
+		}
+	}
+
```

Comment:
Do we really need this?  Can we still proceed if one of the atoms doesn't exist?

Reply:
We can live without atoms; we just won't have the functionality they indicate.

See https://github.com/djpohly/dwl/blob/8b7cc8ada65939b12320b9c77d656401c1ca9f15/dwl.c#L1720

Reply:
Okay, I have removed those checks. I just felt _unsafe_ without checking for potential error values.